lib/openhab/dsl/items/location_item.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'java'
+require 'forwardable'
+require 'openhab/dsl/items/item_command'
+require 'openhab/dsl/items/item_delegate'
+
+module OpenHAB
+  module DSL
+    module Items
+      #
+      # Delegator to OpenHAB Player Item
+      #
+      class LocationItem
+        extend OpenHAB::DSL::Items::ItemCommand
+        extend OpenHAB::DSL::Items::ItemDelegate
+        extend Forwardable
+
+        def_item_delegator :@location_item
+
+        def_oh_item_delegator :@location_item, :distance_from
+
+        # Alias '-' to distance_from
+        alias :- distance_from
```

Comment:
The mathematician in me doesn't really like this, since generally:
```
a - b != b - a unless a == b
```
And that rule breaks here.

Reply:
I can see that.  It did seem to flow to think that you are substracing for the distance from two points.  

Do we want to take it out?  I need to think if in ruby there are other places like when you subtract arrays that don't abide by that rule.

Reply:
I think it can be fine to leave it, it's unlikely anyone would use it with the intent of getting a negative number.